templates/default/nodewatcher.cfg.erb

```diff
@@ -1,4 +1,5 @@
 [nodewatcher]
 region = <%= node['cfncluster']['cfn_region'] %>
 scheduler = <%= node['cfncluster']['cfn_scheduler'] %>
-proxy = <%= node['cfncluster']['cfn_proxy'] %>
\ No newline at end of file
+proxy = <%= node['cfncluster']['cfn_proxy'] %>
+scale_down_idle_time = <%= node['cfncluster']['cfn_scale_down_idle_time'] %>
```

Comment:
If it's not annoying and it's coherent with the rest of the parameters names I'd go for using the name `cfn_scaledown_idletime`

Reply:
I wanted to avoid that since scaledown is not one word, but I can accommodate that change. It would have to be changed in every repository.